src/preload/preload.js

```diff
@@ -1,141 +1,386 @@
-// This adds the "preload" extension to htmx.  By default, this will
-// preload the targets of any tags with `href` or `hx-get` attributes
-// if they also have a `preload` attribute as well.  See documentation
-// for more details
-htmx.defineExtension('preload', {
+/**
+ * This adds the "preload" extension to htmx. The extension will 
+  * preload the targets of elements with "preload" attribute if:
+  * - they also have `href`, `hx-get` or `data-hx-get` attributes
+  * - they are radio buttons, checkboxes, select elements and submit
+  *   buttons of forms with `method="get"` or `hx-get` attributes
+  * The extension relies on browser cache and for it to work
+  * server response must include `Cache-Control` header
+  * e.g. `Cache-Control: private, max-age=60`.
+  * For more details @see https://v2-0v2-0.htmx.org/extensions/preload/
```

Comment:
https://v2-0v2-0.htmx.org/extensions/preload/ should be https://htmx.org/extensions/preload/

The v2 sub site was just a snapshot of the main htmx.org site that was used during v2 development and early publication but the main site now has all the current v2 documentation.

Reply:
Thank you @MichaelWest22, I have updated the link.